spec/controller/api/wrap_parameters_spec.rb

```diff
@@ -0,0 +1,319 @@
+          initial_params = {root: :value, param: :value}
+          expected_result = {root: :value, param: :value}
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+    end
+
+    context 'when parameters wrapper is declared with :include option' do
+      context 'and :include option is set as Symbol' do
+        let(:controller) do
+          Class.new(RageController::API) do
+            wrap_parameters :root, include: :param_a
+
+            def index
+              render json: params
+            end
+          end
+        end
+
+        it 'wraps the param that is set to be included' do
+          initial_params = {param_a: :value, param_b: :value}
+          expected_result = {param_a: :value, param_b: :value, root: {param_a: :value}}
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+
+      context 'and :include option is set as Array' do
+        let(:controller) do
+          Class.new(RageController::API) do
+            wrap_parameters :root, include: [:param_a, :param_b]
+
+            def index
+              render json: params
+            end
+          end
+        end
+
+        it 'wraps the params that are set to be included' do
+          initial_params = {param_a: :value, param_b: :value, param_c: :value}
+          expected_result = {
+            param_a: :value,
+            param_b: :value,
+            param_c: :value,
+            root: {param_a: :value, param_b: :value}
+          }
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+    end
+
+    context 'when parameters wrapper is declared with :exclude option' do
+      context 'and :exclude option is set as Symbol' do
+        let(:controller) do
+          Class.new(RageController::API) do
+            wrap_parameters :root, exclude: :param_a
+
+            def index
+              render json: params
+            end
+          end
+        end
+
+        it 'wraps the params except the param that is set to be excluded' do
+          initial_params = {param_a: :value, param_b: :value}
+          expected_result = {param_a: :value, param_b: :value, root: {param_b: :value}}
+
+          response = run_action(controller, :index, params: initial_params, env: {'CONTENT_TYPE' => "application/json"})
+          expect(response).to match([200, instance_of(Hash), [expected_result.to_json]])
+        end
+      end
+
+      context 'and :exclude option is set as Array' do
```

Comment:
Could you please also add a test case for when the `:exclude` option is not present in the params?